src/main/core/main.c

```diff
@@ -59,6 +59,22 @@ static void _main_logEnvironment(gchar** argv, gchar** envv) {
     }
 }
 
+static int _raise_rlimit(int resource) {
+    struct rlimit lim = {0};
+    if (getrlimit(resource, &lim) != 0) {
+        error("Could not get rlimit: %s", strerror(errno));
```

Comment:
Could we add the type of rlimit (files or procs, etc) that we are failing to raise? Ideally a string but even the int would be better than nothing.

Reply:
Added.